### DISCUSSION TITLE
How do you measure exactly 45 minutes?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#463] [Galaxy Trivia] [answered]
You have two ropes. Each burns completely in exactly one hour. However, they are different lengths and thus burn at different rates. Also, along each rope the width & material changes, so they burn at different rates at different places. The only thing you know is that they each burn completely in one hour exactly. You have a lighter.

**Comment by ParanoidUser** [comment]
@wfrsk, @DrakonZZZ, @Taky0nDon, @JulioG516, any idea? 🕯️

**Comment by peterbaccaro** [comment, marked as answer]
Light one of the ropes on fire on both ends and light the second rope on one end at the same time. 
When the first rope burns out, 30 minutes have elapsed. At that exact moment, you light the unlit end of the second rope.
Because 30 minutes of the second rope have already been used up, 30 more remain. 
Lighting the other end at the moment the first rope burns up will cause the remaining part of the second rope to burn up in 15 minutes. Once the second rope has been consumed by the flames, exactly 45 minutes have passed.
